lib/rss/image.rb

```diff
@@ -23,7 +23,7 @@ module RSS
 
   module ImageModelUtils
     def validate_one_tag_name(ignore_unknown_element, name, tags)
-      if !ignore_unknown_element
+      unless ignore_unknown_element
```

Comment:
```suggestion
      if !ignore_unknown_element
```

Not in `Style/StringLiterals`.